src/main/webapp/i18n/en/lectureUnit.json

```diff
@@ -13,10 +13,47 @@
             },
             "releaseDate": "Release Date"
         },
+        "videoUnit": {
+            "created": "A new video unit created",
+            "updated": "Video unit updated",
+            "tooltip": "This is a Video Unit.",
+            "description": "Description",
+            "notReleasedTooltip": "Video only visible to teaching assistants and instructors. Release date:",
+            "youTube": {
+                "errorCode2": "The request contains an invalid parameter value. For example, this error occurs if you specify a video ID that does not have 11 characters, or if the video ID contains invalid characters.",
+                "errorCode5": " The requested content cannot be played in an HTML5 player or another error related to the HTML5 player has occurred.",
+                "errorCode100": "The video requested was not found. This error occurs when a video has been removed (for any reason) or has been marked as private.",
```

Comment:
```suggestion
                "errorCode100": "The requested video was not found. This error occurs when a video has been removed (for any reason) or has been marked as private.",
```

Reply:
Not changed to be consistent with text from https://developers.google.com/youtube/iframe_api_reference